programs-service/src/validation/baseValidator.js

```diff
@@ -0,0 +1,30 @@
+const {Validation, Sanitation} = require('@utils/');
+const {ValidationError} = require('@validationErrors/');
+const Ajv = require('ajv');
+const setupAsync = require('ajv-async');
+
+const ajv = setupAsync(new Ajv({allErrors: true}));
+
+/*// TODO: Natalie - function that returns function?
+const rewriteValidationError = error =>
+    error.errors.map(err => {
+        return `${err.dataPath.substring(1)} ${err.message}`;
+    });*/
+
+const validate = schema => {
+    const validateBySchema = ajv.compile(schema);
+
+    return (req, res, next) => {
```

Comment:
as far as I know ajv does that by itself (removing the props which failed the validation)

Reply:
I think we can return to this matter later when there's a working database, however you're welcomed to collaborate at any time :)